packages/components/pie-webc-core/src/decorators/valid-property-values.ts

```diff
@@ -25,8 +23,7 @@ export const validPropertyValues = <T>(componentName: string, validValues: reado
             } else {
                 this[privatePropertyKey] = value;
             }
-
-            this.requestUpdate(propertyKey, oldValue);
         },
+        configurable: true,
```

Comment:
Well done for doing this – said to Ben I'd look at it when I got back, but job done! 🎉 🎉 🎉